nexteuropa_poetry.module

```diff
@@ -51,6 +51,7 @@ function nexteuropa_poetry_client() {
 
   $log_level = variable_get('nexteuropa_poetry_log_level', 'none');
   $log_level = ($log_level === 'none') ? FALSE : $log_level;
+  $poetry_service = variable_get('poetry_service', array('address' => ''));
```

Comment:
We should indeed add the service WSDL as a global setting in the settings page, we could report below the text field what written on the [client documentation](https://github.com/ec-europa/oe-poetry-client/blob/master/docs/01-poetry.md#configuration):

```
Set the following values for service.wsdl according to your environment:
- Production: http://intragate.ec.europa.eu/DGT/poetry_services/components/poetry.cfc?wsdl
- Testing: http://intragate.test.ec.europa.eu/DGT/poetry_services/components/poetry.cfc?wsdl
``` 

Add the new configuration in this [file here](https://github.com/ec-europa/nexteuropa_poetry/blob/master/includes/nexteuropa_poetry.admin.inc) and add also a test in [settings.feature](https://github.com/ec-europa/nexteuropa_poetry/blob/master/tests/features/settings.feature).

Also, until this PR https://github.com/InterNations/http-mock/pull/42 is not merged in let's comment `5.6` from travis.yml so that tests temporarely run only on php 7. Also change in .drone.yml `fpfis/php56-dev` into `fpfis/php71-dev` for the same reason.

Reply:
I've done the requested changes. I've put the values for service.wsdl in the README.md file.